python-example/README.md

```diff
@@ -210,7 +216,8 @@ Our next step is to start our application:
 python3 server.py
 ```
 
-All that is left for us to do at this point is to visit [localhost](http://localhost)/[exception](http://localhost/exception) and refresh the page, triggering
-our app to generate and emit a trace of that transaction. When the trace data collected from the OpenTelemetry collector
-are ingested, you can examine them in the Tanzu Observability user interface.
+All that is left for us to do at this point is to visit [localhost](http://localhost)
```

Comment:
I think instead of [localhost](http://localhost) we should display these URLs (like http://localhost) for better clarity - for instance the .NET example uses a custom port which isn't displayed in the link text, which could be confusing.

Reply:
Done :)